### DISCUSSION TITLE
Community Group on LinkedIn

### DISCUSSION THREAD

**Discussion by bitscoper** [ultralytics/ultralytics#9904] [General]
Let's connect, inspire, and explore together on LinkedIn.
[https://linkedin.com/groups/13009947/](https://www.linkedin.com/groups/13009947/)

**Comment by glenn-jocher** [comment]
Thank you for sharing this! Connecting with others in the community can foster learning and collaboration. 🤝 For technical discussions or specific issues, GitHub remains the ideal platform, but it's great to have a space on LinkedIn for broader conversations and networking. See you there!
